spec/unit/filecache_spec.rb

```diff
@@ -84,6 +84,24 @@ def write_file(file, contents)
         end
       end
 
+      it "sets the encoding to UTF-8 when reading a file" do
+        Dir.mktmpdir do |dir|
+          file = File.join(dir, "testing")
+          write_file(file, "my data")
+          File.expects(:read).with(file, :encoding => 'BOM|UTF-8')
+          @cache.read_file(file)
+        end
+      end
+
+      it "reads a file with unicode characters" do
+        Dir.mktmpdir do |dir|
+          file = File.join(dir, "testing")
+          write_file(file, "☃")
```

Comment:
👍  will update. 

the ruby 1.9 cell in travis failed with an error similar to what you describe:
```
$ bundle exec rake $CHECK
rspec spec
/home/travis/.rvm/gems/ruby-1.9.3-p551/gems/rspec-core-3.5.2/lib/rspec/core/configuration.rb:1435:in `load': /home/travis/build/puppetlabs/hiera/spec/unit/filecache_spec.rb:99: invalid multibyte char (US-ASCII) (SyntaxError)
/home/travis/build/puppetlabs/hiera/spec/unit/filecache_spec.rb:99: invalid multibyte char (US-ASCII)
/home/travis/build/puppetlabs/hiera/spec/unit/filecache_spec.rb:99: syntax error, unexpected $end, expecting ')'
          write_file(file, "☃")
                              ^
	from /home/travis/.rvm/gems/ruby-1.9.3-p551/gems/rspec-core-3.5.2/lib/rspec/core/configuration.rb:1435:in `block in load_spec_files'
	from /home/travis/.rvm/gems/ruby-1.9.3-p551/gems/rspec-core-3.5.2/lib/rspec/core/configuration.rb:1433:in `each'
	from /home/travis/.rvm/gems/ruby-1.9.3-p551/gems/rspec-core-3.5.2/lib/rspec/core/configuration.rb:1433:in `load_spec_files'
	from /home/travis/.rvm/gems/ruby-1.9.3-p551/gems/rspec-core-3.5.2/lib/rspec/core/runner.rb:100:in `setup'
	from /home/travis/.rvm/gems/ruby-1.9.3-p551/gems/rspec-core-3.5.2/lib/rspec/core/runner.rb:86:in `run'
	from /home/travis/.rvm/gems/ruby-1.9.3-p551/gems/rspec-core-3.5.2/lib/rspec/core/runner.rb:71:in `run'
	from /home/travis/.rvm/gems/ruby-1.9.3-p551/gems/rspec-core-3.5.2/lib/rspec/core/runner.rb:45:in `invoke'
	from /home/travis/.rvm/gems/ruby-1.9.3-p551/gems/rspec-core-3.5.2/exe/rspec:4:in `<top (required)>'
	from /home/travis/.rvm/gems/ruby-1.9.3-p551/bin/rspec:23:in `load'
	from /home/travis/.rvm/gems/ruby-1.9.3-p551/bin/rspec:23:in `<main>'
	from /home/travis/.rvm/gems/ruby-1.9.3-p551/bin/ruby_executable_hooks:15:in `eval'
	from /home/travis/.rvm/gems/ruby-1.9.3-p551/bin/ruby_executable_hooks:15:in `<main>'
rake aborted!
Command failed with status (1): [rspec spec...]
/home/travis/build/puppetlabs/hiera/Rakefile:52:in `block in <top (required)>'
/home/travis/.rvm/gems/ruby-1.9.3-p551/bin/ruby_executable_hooks:15:in `eval'
/home/travis/.rvm/gems/ruby-1.9.3-p551/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => spec
(See full trace by running task with --trace)
The command "bundle exec rake $CHECK" exited with 1.
Done. Your build exited with 1.
```